storagenode/peer.go

```diff
@@ -110,6 +114,9 @@ type Peer struct {
 		Sender    *orders.Sender
 	}
 
+	//TODO: Where to put this? Storage2/Kademlia?
```

Comment:
What do you mean?

Reply:
Other services have been compartmentalized into the Storage2 and Kademlia structs, but Vouchers don't really fall into either of those so, I could probably get rid of this comment

Reply:
ok yeah even though vouchers are tied to kad, it's kind of its own thing